### DISCUSSION TITLE
How do I make the drop box wider (MRS)? HTML Notepad. Please Help

### DISCUSSION THREAD

**Discussion by youwann7** [community/community#21366] [Other Feature Feedback, Questions, & Ideas]
<p>&lt;!DOCTYPE html&gt;<br>
&lt;html&gt;<br>
&lt;head&gt;<br>
&lt;meta name=“viewport” content=“width=10px, initial-scale=10000”&gt;<br>
&lt;style&gt;<br>
body {font-family: Arial;}</p>
<p>/* Style the tab */<br>
.tab {<br>
overflow: hidden;<br>
border: 912x solid <span class="hashtag">#ccc</span>;<br>
background-color: <span class="hashtag">#f1f1f1</span>;<br>
}</p>
<p>/* Style the buttons inside the tab */<br>
.tab button {<br>
background-color: inherit;<br>
float: left;<br>
border: none;<br>
outline: none;<br>
cursor: pointer;<br>
padding: 14px 16px;<br>
transition: 0.3s;<br>
font-size: 17px;<br>
}</p>
<p>/* Change background color of buttons on hover */<br>
.tab button:hover {<br>
background-color: <span class="hashtag">#ddd</span>;<br>
}</p>
<p>/* Create an active/current tablink class */<br>
.tab button.active {<br>
background-color: <span class="hashtag">#ccc</span>;</p>
<p>}</p>
<p>/* Style the tab content */<br>
.tabcontent {<br>
display: none;<br>
padding: 6px 12px;<br>
border: 3px solid <span class="hashtag">#ccc</span>;<br>
border-top: none;<br>
}<br>
&lt;/style&gt;<br>
&lt;/head&gt;<br>
&lt;body&gt;</p>
<p>&lt;p&gt;Click on the buttons inside the tabbed menu:&lt;/p&gt;</p>
<p>&lt;div class=“tab”&gt;<br>
&lt;button class=“tablinks” onclick=“openCity(event, ‘Client Details’)”&gt;Client Details&lt;/button&gt;<br>
&lt;button class=“tablinks” onclick=“openCity(event, ‘Notes’)”&gt;Notes&lt;/button&gt;<br>
&lt;/div&gt;</p>
<p>&lt;div id=“Client Details” class=“tabcontent”&gt;<br>
&lt;h3&gt;Personal Information&lt;/h3&gt;<br>
&lt;p&gt;&lt;!DOCTYPE html&gt;<br>
&lt;html&gt;<br>
&lt;body&gt;</p>
<p>&lt;!DOCTYPE html&gt;<br>
&lt;html lang = “en”&gt;<br>
&lt;head&gt;<br>
&lt;meta charset = “UTF-8” /&gt;<br>
&lt;/head&gt;<br>
&lt;form&gt;<br>
&lt;fieldset&gt;<br>
&lt;p&gt;<br>
&lt;label&gt;Title*&lt;/label&gt;<br>
&lt;select id = “myList”&gt;<br>
&lt;option value = “Miss”&gt;Miss&lt;/option&gt;<br>
&lt;option value = “Mrs”&gt;Mrs&lt;/option&gt;<br>
&lt;option value = “Mr”&gt;Mr&lt;/option&gt;<br>
&lt;/select&gt;<br>
&lt;/p&gt;<br>
&lt;/fieldset&gt;<br>
&lt;/form&gt;<br>
&lt;/body&gt;<br>
&lt;/html&gt;</p>
<p>&lt;/body&gt;<br>
&lt;/html&gt;</p>
<p>&lt;/body&gt;<br>
&lt;/html&gt;<br>
.&lt;/p&gt;<br>
&lt;/div&gt;</p>
<p>&lt;div id=“Notes” class=“tabcontent”&gt;<br>
&lt;h3&gt;Notes&lt;/h3&gt;<br>
&lt;p&gt;.&lt;/p&gt;<br>
&lt;/div&gt;</p>
<p>&lt;script&gt;<br>
function openCity(evt, cityName) {<br>
var i, tabcontent, tablinks;<br>
tabcontent = document.getElementsByClassName(“tabcontent”);<br>
for (i = 0; i &lt; tabcontent.length; i++) {<br>
tabcontent[i].style.display = “none”;<br>
}<br>
tablinks = document.getElementsByClassName(“tablinks”);<br>
for (i = 0; i &lt; tablinks.length; i++) {<br>
tablinks[i].className = tablinks[i].className.replace(" active", “”);<br>
}<br>
document.getElementById(cityName).style.display = “block”;<br>
evt.currentTarget.className += " active";<br>
}<br>
&lt;/script&gt;</p>
<p>&lt;/body&gt;<br>
&lt;/html&gt; </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/48d448fa9183f77bcb73ae47498e2eeb17279862.png" data-download-href="/uploads/short-url/aohcbw2uKzUUUGyhruFnL2eMN10.png?dl=1" title="Capture.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/48053770/181063892-97a0843a-4be6-44d9-b853-d9e7cc1add0f.png" alt="Capture.PNG" data-base62-sha1="aohcbw2uKzUUUGyhruFnL2eMN10" width="690" height="323" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/48d448fa9183f77bcb73ae47498e2eeb17279862_2_690x323.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/48d448fa9183f77bcb73ae47498e2eeb17279862_2_1035x484.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/4/48d448fa9183f77bcb73ae47498e2eeb17279862_2_1380x646.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/48d448fa9183f77bcb73ae47498e2eeb17279862_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.PNG</span><span class="informations">1913×898 5.82 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/youwann7">@youwann7</a>!</p>
<p>I’ve moved your post into the Programming Help and Discussion board as I think you’ll probably get better answers to your questions here.</p>
<p>To answer your question, you should be able to use the <strong>width</strong> CSS attribute on <strong><span class="hashtag">#myList</span></strong>  to change the width of the dropdown menu. However, every browser interprets the display styles for dropdown menus differently, so you might have to fiddle with the values and selectors. For more information, I personally recommend the CSS Tricks blog which has an article all about <a href="https://css-tricks.com/dropdown-default-styling/" rel="nofollow noopener">styling dropdown menus</a>.</p>
<p>Hope this helps!</p>

**Comment by hassan647** [comment]
<p>just increase the width of your dropbox</p>
